testsettings.py

```diff
@@ -15,6 +15,7 @@
 NOSE_ARGS = [
     #'--no-migrations' # trim ~120s from test run with db tests
     #'--with-fixture-bundling',
+    '--with-doctest',
```

Comment:
There weren't that many if I remember correctly. I think I decided not to do that because the scope of work of getting switched to nose was already pretty overwhelming and rewriting doctests didn't seem an overly high priority. Some of them were broken because they had not been running anyway due to Django's fragile and error prone mechanism for registering them to be run.